minimal/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "minimal",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "apollo-cache-inmemory": "^1.1.7",
+    "apollo-client": "^2.2.3",
+    "apollo-client-preset": "^1.0.8",
```

Comment:
If you use `apollo-client-preset`, there is no need to add `apollo-cache-inmemory`, `apollo-client`, and `apollo-link-http`. If you want to install those three dependencies, then you can remove the `apollo-client-preset` dependency.